DSharpPlus.CommandsNext/Attributes/RequireRolesAttribute.cs

```diff
@@ -73,6 +76,11 @@ public enum RoleCheckMode
         /// </summary>
         All,
 
+        /// <summary>
+        /// Member is required to have exactly the same roles as specified; no extra roles may be present.
+        /// </summary>
+        Pure,
```

Comment:
This should be renamed to `SpecifiedOnly` given what it does then.

Reply:
Done :ok_hand: